setupTests.base.ts

```diff
@@ -10,11 +10,13 @@ beforeEach(() => {
   jest.spyOn(console, 'error').mockImplementation((...args) => {
     isConsoleErrorOrWarning = true
     originalError(...args)
+    console.log('ERROR', ...args)
```

Comment:
Why do we need it? We already do console.error and console.warn via originalError and originalWarn?